tools/windows-node-installer/test/e2e/azure_test.go

```diff
@@ -106,6 +105,7 @@ func TestCreateVM(t *testing.T) {
 	t.Run("check if required security rules are present", testRequiredRules)
 	t.Run("check if ansible is able to ping on the WinRmHttps port", testAnsiblePing)
 	t.Run("check if container logs port is open in Windows firewall", testAzureInstancesFirewallRule)
+	t.Run("check if SSH connection is available", testAzureSSHConnection)
```

Comment:
The e2e test should be part of the commit where you are introducing the feature.